### PR TITLE
ci(evergreen): use docker compose where available instead of standalone docker-compose

### PR DESCRIPTION
**Pull request by gribnoysup**
Will not help us now, but should make figuring these errors easier in the future

### CODE REVIEW

**Review by addaleax — APPROVED** [review]

**Comment by gribnoysup** [comment]
@addaleax I did a bunch of changes here after you approved, and merged when CI was green to unblock main. If you want to and have time, maybe you want to take a look at my scrappy bash script writing, if there is a better way to do this check, I'm happe to open a follow-up PR for that 😄

**Pull request merged**
